src/components/AboutPage.js

```diff
@@ -1,15 +1,140 @@
 import React from 'react';
+import '../styling/about.css';
```

Comment:
all style sheets should be imported into one place in index.css
https://github.com/nupur-n/build4blm/blob/dev/src/index.css#L1

Reply:
i think i need to merge dev into this branch. my index.css seems outdated

Reply:
done. question: in about.css, i specify some things for elements (i.e. hr, p, ul) rather than classes. should I modify to use classes instead to avoid conflicts with other pages' styling?